### PR TITLE
Docs: Add new plugins options page

### PR DESCRIPTION
**Pull request by dgault**
This is a follow up to https://github.com/openmicroscopy/bioformats/pull/2891

Adds a new docs page for configuring of the Bio-Formats plugin, namely the slice label pattern.

### CODE REVIEW

**Comment by sbesson** [comment]
The new page also needs to be added to the `toctree` of the top-level `users/index.rst` file.

File: docs/sphinx/users/imagej/options.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+configuration of features for the plugin such as upgrade checking or modifying 
+the slice label pattern.
+
+Configuring the slice label pattern
+-----------------------------------
+
+The slice label is the text displayed at the top of the image window in ImageJ.
+The values displayed here can be modified by configuring the slice label pattern.
+
+The list of available parameters for configuration is as follows:
```
Comment (rleigh-codelibre) [inline docs/sphinx/users/imagej/options.rst]: You can just use `follows::` here and then you don't need `::` on the next line.

File: docs/sphinx/users/imagej/options.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+::
+
+    %s - series index
```
Comment (rleigh-codelibre) [inline docs/sphinx/users/imagej/options.rst]: This is fine, but you could use a definition list here, i.e.

```
%n
  series name
%c
  channel index
etc.
```

**Comment by rleigh-codelibre** [comment]
Overall this looks fine; some small Sphinx markup improvements noted, but they aren't required.

File: docs/sphinx/users/imagej/options.rst (hunk 2 above)
Comment (dgault) [reply docs/sphinx/users/imagej/options.rst]: Ive updated and opted for a field list which seemed to look better from a formatting perspective but I can swap to a definition or option list if that is more the correct implementation.

**Pull request merged**